elements/pfe-readtime/src/pfe-readtime.js

```diff
@@ -0,0 +1,214 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeReadtime extends PFElement {
+  static get tag() {
+    return "pfe-readtime";
+  }
+
+  static get meta() {
+    return {
+      title: "Readtime",
+      description:
+        "This element will collect a word count on a given section and calculate the readtime based on that count."
+    };
+  }
+
+  get templateUrl() {
+    return "pfe-readtime.html";
+  }
+
+  get styleUrl() {
+    return "pfe-readtime.css";
+  }
+
+  // static get events() {
+  //   return {
+  //   };
+  // }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
+  static get properties() {
+    return {
+      wordCount: {
+        title: "Word count",
+        type: Number,
+        observer: "_wordCountChangeHandler"
+      },
+      wpm: {
+        title: "Words per minute",
+        type: Number,
+        observer: "_WPMChangeHandler"
+      },
+      for: {
+        title: "For",
+        //This is the uniqueId of the target
+        type: String,
+        observer: "_forChangedHandler"
+      },
+      readtime: {
+        title: "Readtime",
+        type: String,
+        observer: "_readTimeChangehandler"
+      }
+    };
+  }
+
+  static get slots() {
+    return {};
+  }
+
+  constructor() {
+    PFElement._debugLog = true;
+    super(PfeReadtime, { type: PfeReadtime.PfeType, delayRender: true });
+
+    this.readStringTemplate = "%t-minute read";
```

Comment:
Maybe add a comment here describing this is the default value?

Reply:
I'm not sure if this code suggestion is exactly what is needed, but it should only be replacing the token `%t` with the number and not the whole string
```suggestion
    this.readStringTemplate = "%t";
```